vulkaninfo/vulkaninfo.cpp

```diff
@@ -951,13 +964,11 @@ PrinterCreateDetails get_printer_create_details(ParsedResults &parse_data, AppIn
         case (OutputCategory::html):
             create.output_type = OutputType::html;
             break;
-        case (OutputCategory::devsim_json):
+        case (OutputCategory::profile_json):
             create.output_type = OutputType::json;
-            create.start_string = std::string("{\n\t\"$schema\": \"https://schema.khronos.org/vulkan/devsim_1_0_0.json#\",\n") +
-                                  "\t\"comments\": {\n\t\t\"desc\": \"JSON configuration file describing GPU " +
-                                  std::to_string(parse_data.selected_gpu) + " (" + selected_gpu.props.deviceName +
-                                  "). Generated using the " + executable_name + " program.\",\n\t\t\"vulkanApiVersion\": \"" +
-                                  VkVersionString(inst.vk_version) + "\"\n" + "\t}";
+            create.start_string =
+                "{\n\t\"$schema\": "
+                "\"https://raw.githubusercontent.com/KhronosGroup/Vulkan-Profiles/master/schema/profile_schema.json\"";
```

Comment:
The profile file is not generated for any version of the profile schema but for the profile schema API version that matches a Vulkan Header revision ideally.

This allows avoiding to require Vulkan capabilities that didn't exist for this Vulkan Header revision.

Reply:
So this should be changed such that the schema used has the same API version as the device?
Also is there going to be an official location for these schema files? IE currently i'm referencing this repo but that doesn't seem very official.

Reply:
Yes same API version than the device.

The official location: https://schema.khronos.org/vulkan/

Reply:
I can't use the same schema as the device version because it may not be on the website.
For example, MoltenVK uses 1.1.304.